hiba-chk.c

```diff
@@ -39,6 +40,10 @@ check_access(const struct hibaenv *env, const struct hibacert *cert, const char
 	int len;
 	int verdict = HIBA_CHECK_NOGRANTS;
 	struct hibaext **grants;
+	bool writed = false;
```

Comment:
These are not acheiving anything since you never check the value of `writed` or `writed_groups`